### DISCUSSION TITLE
input bam file to createArrowFiles

### DISCUSSION THREAD

**Discussion by yingyuan830** [GreenleafLab/ArchR#1565] [Questions / Documentation]
I'd like to use bam file as input file for the createArrowFiles function, but there are many errors, could you please give a specific example for it ? Should the bam file be sorted ahead ? and is there any function can be used to remove duplicates from the file ? Thanks!

**Comment by rcorces** [comment]
This will eventually be shown in an upcoming release of the documentation but I do not have bandwidth to provide this now. you can search old issues for details as they have been provided previously.
for ex. https://github.com/GreenleafLab/ArchR/issues/589
